app/server.js

```diff
@@ -13,7 +13,7 @@ const apiDoc = require('./api-v3/api-doc')
 const apiService = require('./api-v3/services/apiService')
```

Comment:
In this file we instantiate:

```js
securityHandlers: {
  bearerAuth: (req) => {
    return verifyJwks(req.headers['authorization'])
  },
}
```

This probably shouldn't even be set as an option if `AUTH_TYPE` is `NONE`

Reply:
done